components/identity-mgt/org.wso2.carbon.identity.mgt.ui/src/main/resources/web/identity-mgt/update_credential.jsp

```diff
@@ -212,7 +212,7 @@
         </form>
     </div>
     <script type="text/javascript">
-        showCaptcha('<%=captchaImageUrl%>');
+        showCaptcha('<%=Encode.forJavaScriptAttribute(captchaImageUrl)%>');
```

Comment:
This should be Encode.forJavascritptBlock since the function was invoked within a script block